### PR TITLE
CON-3107 Tests(DPxAI): upload test for quest02 colorful-legs

### PR DESCRIPTION
**Pull request by lotoussa**
[CON-3107](https://01talent.atlassian.net/browse/CON-3107)

[CON-3107]: https://01talent.atlassian.net/browse/CON-3107?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: dom/colorful-legs_test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  // Get the new colors of both legs
+  const legLeftColor = await page.$eval('#leg-left', (node) => getComputedStyle(node).backgroundColor)
+  const legRightColor = await page.$eval('#leg-right', (node) => getComputedStyle(node).backgroundColor)
+
+  // Check if both legs have been assigned the same new color
+  eq(legLeftColor, legRightColor)
+
+  // Ensure the new color is not black
+  eq(legLeftColor !== 'rgb(0, 0, 0)', true, 'The color of the legs should not be black')
```
Comment (nprimo) [inline dom/colorful-legs_test.js:16]: There is nothing in the subject that mentions that black is not a valid new color

Comment (nprimo) [inline dom/colorful-legs_test.js:16]: This check get "triggered" (and fails the solutions) even with this solutions:
```js
// ...
const handleChangeLegColor = event => {
  const randomColor = `#${Math.floor(Math.random() * 16777215).toString(16)}`
  legRight.style.backgroundColor = 'red'
  legLeft.style.backgroundColor = randomColor
}
// ...
```

Comment (lotoussa) [reply dom/colorful-legs_test.js]: right, i deleted the test
black isn't a color so the generative IA helping me building the test for this exercice appears coherent to me and i kept this before your input

**Review by Oumaimafisaoui — APPROVED** [review]
LGTM

**Pull request merged**